src/js/components/AppVersionComponent.jsx

```diff
@@ -187,8 +188,15 @@ var AppVersionComponent = React.createClass({
 
     var urisNode = (appVersion.uris == null || appVersion.uris.length === 0)
       ? <UnspecifiedNodeComponent />
-      : appVersion.uris.map(function (u) {
-        return <dd key={u}>{u}</dd>;
+      : appVersion.uris.map(function (uri, i) {
+        var parsedURI = url.parse(uri);
+        var linkName = uri;
```

Comment:
Both not ideal, yeah.

In one case it's only a string -> name.
In the other case it's an anchor -> node.

How about ```linkNameNodeThatContainsAnUriMaybeWithAnchor``` ?

But I tend to just ```linkNode```, will change it.